requirements.txt

```diff
@@ -35,12 +34,12 @@ regex==2021.4.4
 requests==2.25.1
 resampy==0.2.2
 scikit-learn==0.24.2
-scipy==1.6.2
+scipy==1.5.4
```

Comment:
did you change the version of `scipy`? if so, why?

Reply:
Actually @jbitton  When I was installing these requirements in my local system then it will throw an error. Especially in `scipy`

That's why I changed !

Reply:
You can see

![scipy](https://user-images.githubusercontent.com/56601466/123369280-1a21c500-d59b-11eb-860f-9654cc110118.PNG)

Reply:
@tanujdhiman can you try upgrading pip before installing the requirements, and see if you still get this error with our version of scipy? Like this: `pip install -U pip`

Reply:
@tanujdhiman can you resolve this please? Have you made progress on it?

Reply:
@tanujdhiman please update on your progress on this, and/or just remove this change. We don't want to change our requirements unless necessary.